vllm/worker/model_runner.py

```diff
@@ -1262,7 +1262,7 @@ def capture_model(self, kv_caches: List[List[torch.Tensor]]) -> None:
                                 self.parallel_config))
                         num_kv_heads = self.model_config.get_num_kv_heads(
                             self.parallel_config)
-                        if num_qo_heads // num_kv_heads >= 4:
+                        if (num_qo_heads // num_kv_heads) not in [1, 2, 4, 8]:
```

Comment:
```suggestion
                        if (num_qo_heads // num_kv_heads) not in (1, 2, 4, 8):
```
Also cc @LiuXiaoxuanPKU @yzh119